public/datasets/zoo.json

```diff
@@ -0,0 +1,113 @@
+{
+  "fields": [
+    {
+      "type": "categorical",
+      "id": "Animal Name",
+      "description": "Name of the animal"
+    },
+    {
+      "type": "categorical",
+      "id": "Hair",
+      "description": "Does the animal have hair?"
+    },
+    {
+      "type": "categorical",
+      "id": "Feathers",
+      "description": "Does the animal have feathers?"
+    },
+    {
+      "type": "categorical",
+      "id": "Eggs",
+      "description": "Does the animal lay eggs?"
+    },
+    {
+      "type": "categorical",
+      "id": "Milk",
+      "description": "Does the animal produce milk?"
+    },
+    {
+      "type": "categorical",
+      "id": "Airborne",
+      "description": "Does the animal fly?"
+    },
+    {
+      "type": "categorical",
+      "id": "Aquatic",
+      "description": "Does the animal live in water?"
+    },
+    {
+      "type": "categorical",
+      "id": "Predator",
+      "description": "Is the animal a predator?"
+    },
+    {
+      "type": "categorical",
+      "id": "Toothed",
+      "description": "Does the animal have teeth?"
+    },
+    {
+      "type": "categorical",
+      "id": "Backbone",
+      "description": "Does the animal have a vertebrae and backbone?"
+    },
+    {
+      "type": "categorical",
+      "id": "Breathes",
+      "description": "Does the animal breathe?"
+    },
+    {
+      "type": "categorical",
+      "id": "Venomous",
+      "description": "Is the animal venomous?"
+    },
+    {
+      "type": "categorical",
+      "id": "Fins",
+      "description": "Does the animal have fins?"
+    },
+    {
+      "type": "continuous",
+      "id": "Legs",
+      "description": "How many legs does the animal have?"
```

Comment:
I wonder about whether this is really continuous... like, partial legs?  I think of it as categorical: 0-legged, 2-legged, 4-legged, 6-legged, 8-legged.